### DISCUSSION TITLE
Docker is installed but isn't running

### DISCUSSION THREAD

**Discussion by wojtekbogusz** [freedomofpress/dangerzone#836] [Q&A]
hi, not sure this is the place to ask about it. please redirect if not.

i am trying to setup DangerZone on Ubuntu 22.04.
when it starts it display message that "Docker is installed but isn't running. Open Docker and make sure it's running in the background".

however `sudo systemctl status docker` shows "active (running)" and i can even get `docker run hello-world` display "Hello from Docker! This message shows that your installation appears to be working correctly."
what can i do?
thanks!
be well

**Comment by almet** [comment]
Hi @wojtekbogusz, sorry we compeltely missed your question. As I see that you closed the issue I assume that you found a solution? If not, let us know and we'll do our best to help.
